morph.go

```diff
@@ -540,6 +541,7 @@ func getNixContext() *nix.NixContext {
 	return &nix.NixContext{
 		EvalMachines: filepath.Join(assetRoot, "eval-machines.nix"),
 		ShowTrace:    showTrace,
+		KeepGCRoot:   keepGCRoot,
```

Comment:
.. and then de-referenced here, for example.

Reply:
That was what I originally did, and then I moved the star to the `var()` above thinking it would be equivalent and somehow more aesthetic. Apparently it isn't, so I'll investigate.